oarepo_ui/theme/assets/semantic-ui/less/oarepo_ui/themes/default/modules/accordion.variables

```diff
@@ -0,0 +1 @@
+@backgroundColor: #0377cd;
```

Comment:
This should probably stay in docs (if color is related to its branding)

Reply:
@mirekys Hey, the problem is that default background color is white, so the accordion is invisible, so I think better to leave it in oarepo ui, as the user could get annoyed because he is not seeing anything :) This isn't really our branding, I just liked the color that accordions have in RDM so I decided to adopt it as default color for our accordions, as we don't really have colors defined. Is that OK?

Reply:
In that case, you're right & can leave it here :)